src/components/UploadCSV/ErrorModal/ErrorModal.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
+import { Button, ButtonGroup, Text } from '@chakra-ui/react';
+
+import './ErrorModal.css';
+import ErrorIcon from '../../../assets/error.png';
+
+const ErrorModal = ({ CSVFileName, setIsUploadingNewFile, uploadErrors, onEditViewFile }) => {
+  const uploadNewFile = () => {
+    setIsUploadingNewFile(true);
+  };
+
+  const viewFile = e => {
+    setIsUploadingNewFile(false);
+    onEditViewFile(e);
+  };
+
+  return (
+    <div className="error-modal-content">
+      <img src={ErrorIcon} alt="Icon" className="error-modal-icon" />
+      <Text className="error-modal-text">Oops!</Text>
+      <p className="error-modal-text">There was an error with the uploaded file:</p>
+      <p className="error-modal-file-name">{CSVFileName}</p>
+      <div className="error-modal-errors">
+        {uploadErrors.map(error => {
+          const id = uuidv4();
+          return (
+            <p className="error-modal-message" key={id}>
+              *{error}*
+            </p>
+          );
+        })}
```

Comment:
Using `uuid` here is overkill, as the list is static and only renders once. I would use the item index from map:
```
{uploadErrors.map((error, index) => {
// ...
key={index}
```
Libraries like `uuidv4` or `nanoid` should only be used if there isn't a way to assign a unique id to a list of items that will be re-rendered when it is changed. Read more about this [here](https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318).

Reply:
I tried that, but I got an eslint error saying that "Do not use Array index in keys". Should I just disable that error?

Reply:
Disabling that error should be fine in this case.